test/suites/backup.sh

```diff
@@ -104,7 +104,7 @@ test_container_import() {
     lxc start ctImport
     rm "${LXD_DIR}/containers/ctImport"
     if [ "$lxd_backend" != "dir" ] && [ "$lxd_backend" != "btrfs" ]; then
-      rm -rf "${LXD_DIR}/storage-pools/lxdtest-$(basename "${LXD_DIR}")/snapshots/ctImport/snap0"
+      rm -rf "${LXD_DIR}/storage-pools/lxdtest-$(basename "${LXD_DIR}")/containers-snapshots/ctImport/snap0"
```

Comment:
So there need to be tests. There is not a single test where actual snapshots are taken, am I right. So we have actually no way of knowing whether the API works or not.

Reply:
There is now a single test ;)